hacep-core/src/test/java/it/redhat/hacep/cluster/TestContainerUpdate.java

```diff
@@ -279,6 +267,8 @@ public void testFailedUpdateOnAllNodes() throws InterruptedException {
             LOGGER.info("TestFailedUpdate: let's pretend everything is ok");
         }
 
+        /* @todo check this "verify" after the new manual managing for kjar update
```

Comment:
probably implement the TODO and change this test to the new manual suspend/resume ?